ee/hogai/summarizer/test/test_format.py

```diff
@@ -0,0 +1,373 @@
+from posthog.schema import Compare, FunnelStepReference
+from posthog.test.base import BaseTest
+
+from ..format import (
+    _extract_series_label,
+    _format_duration,
+    _format_funnels_results,
+    _format_number,
+    _strip_datetime_seconds,
+    compress_and_format_funnels_results,
+    compress_and_format_retention_results,
+    compress_and_format_trends_results,
+)
+
+
+class TestFormat(BaseTest):
```

Comment:
This suite looks like two combined into one: `TestFormatHelpers` (the private functions) and `TestCompression`